libpod/runtime.go

```diff
@@ -705,12 +706,12 @@ func makeRuntime(runtime *Runtime) (err error) {
 
 		manager, err = lock.NewSHMLockManager(lockPath, runtime.config.NumLocks)
 		if err != nil {
-			return errors.Wrapf(err, "error creating SHM locks for libpod")
+			return err
 		}
 	} else {
 		manager, err = lock.OpenSHMLockManager(lockPath, runtime.config.NumLocks)
 		if err != nil {
-			return errors.Wrapf(err, "error opening libpod SHM locks")
```

Comment:
Just curious why you removed the errors.Wrapf here and above.

Reply:
Underlying library has the same message now.